pkgs/applications/audio/musikcube/default.nix

```diff
@@ -17,13 +17,13 @@
 
 stdenv.mkDerivation rec {
   pname = "musikcube";
-  version = "0.96.2";
+  version = "0.96.4";
 
   src = fetchFromGitHub {
     owner = "clangen";
     repo = pname;
     rev = version;
-    sha256 = "1vvc8295yadprfqpmk9i43ghymgsqxsg8zfx3p5i86a8xrdhd9vn";
+    sha256 = "sha256-DYLLnmRHEK2nJUbJfJUDcA8qHZq+qZpHS5ei8qUGmFA=";
```

Comment:
@ryantm this isn't the type of hash I generally expect to see from the bot... am I wrong about that?

Reply:
I guess that's what `nix build` outputs these days. It should be okay because nixpkgs nix mincer supports reading it.

Reply:
Thanks!